src/components/HelpHoverable.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+  import tippy from "sveltejs-tippy";
+  import InfoIcon from "./icons/InfoIcon.svelte";
+
+  export let content;
+  export let link;
+  export let props = {
```

Comment:
We don't need to export `props` anymore.

```suggestion
  let props = {
```